results/bm25s/0_1_10/model_meta.json

```diff
@@ -0,0 +1 @@
+{"name": "bm25s", "revision": "0_1_10", "release_date": "2024-07-10", "languages": ["eng_Latn"], "n_parameters": null, "memory_usage": null, "max_tokens": null, "embed_dim": null, "license": null, "open_source": true, "similarity_fn_name": null, "framework": [], "loader": "BM25Search"}
```

Comment:
Keep this file or no?

Reply:
I think we can remove it? cc @orionw @KennethEnevoldsen